docs/io.cozy.contacts.md

```diff
@@ -40,3 +40,15 @@ The `io.cozy.contacts` doctype is loosely based on the [vCard RFC](https://tools
   - `url`: {string}
   - `label?`: {string} A user-provided localized type of instance
   - `primary?`: {boolean} Indicates a preferred-use instance
+- `groups`: {array} An array containing all the `io.cozy.contacts.groups` this contact is part of
```

Comment:
It's an array with just the ids of the groups, right?

Reply:
Yes, and I should probably specify that.